LuaSkin/LuaSkin/Skin.m

```diff
@@ -403,6 +410,7 @@ - (int)registerLibrary:(const luaL_Reg *)functions metaFunctions:(const luaL_Reg
 
     NSString *fname = getCallerFileName() ;
 
+    // FIXME: Can this all be replaced just by using the libraryName argument?
```

Comment:
Yes, with one caveat: for now, at least, its possible that libraryName might not have a value for a given library... however since it's being applied to the reftable and for identification during debugging only, I suppose it not being present (i.e. not set) for a given table won't cause a problem.

Reply:
Ok, let's wait on that until CommandPost and you have updated all of your modules to use the new one.